src/main/java/org/apache/maven/report/projectinfo/CiManagementReport.java

```diff
@@ -225,23 +199,29 @@ else if ( isCiManagementSystem( system, "travis" ) )
         }
 
         /**
-         * Checks if a CI management system is bugzilla, continuum...
+         * Search system description.
          *
-         * @return true if the CI management system is bugzilla, continuum..., false otherwise.
+         * @param system a system for description
+         * @return system description from properties
          */
-        private boolean isCiManagementSystem( String system, String actual )
+        private String getIntroForManagementSystem( String system )
         {
             if ( StringUtils.isEmpty( system ) )
             {
-                return false;
+                return getI18nString( "general.intro" );
             }
 
-            if ( StringUtils.isEmpty( actual ) )
+            String systemLowerCase = system.toLowerCase( Locale.ENGLISH );
+
+            for ( String systemName : SYSTEMS )
             {
-                return false;
+                if ( systemLowerCase.startsWith( systemName ) )
```

Comment:
I don't understand why we need start with if we have a symbolic name for each system?

Reply:
It was implemented in this way, probably because user can specific your system like Travis CI or some thing more ... so we check startWith
I don't change this behavior.

Reply:
OK, I see.